js/main.js

```diff
@@ -73,25 +91,50 @@ function renderEntry(entry) {
   var img = document.createElement('img');
   var h2 = document.createElement('h2');
   var p = document.createElement('p');
+  var editIcon = document.createElement('i');
 
   divRow.setAttribute('class', 'row');
-  divRow2.setAttribute('class', 'row');
+  divRow2.setAttribute('class', 'row pos-rel');
   divRow3.setAttribute('class', 'row');
   divColFullColHalf.setAttribute('class', 'column-full column-half mb-2.8');
   divColFullColHalf2.setAttribute('class', 'column-full column-half mb-2.8');
   img.setAttribute('src', entry.photo);
-  h2.setAttribute('class', 'mb-0');
+  img.setAttribute('class', 'entry');
+  h2.setAttribute('class', 'entry mb-0');
+  editIcon.setAttribute('class', 'fas fa-pen purple');
+
+  img.setAttribute('data-entry-id', entry.entryId);
+  h2.setAttribute('data-entry-id', entry.entryId);
+  p.setAttribute('data-entry-id', entry.entryId);
+  editIcon.setAttribute('data-entry-id', entry.entryId);
+  divRow.setAttribute('data-entry-id', entry.entryId);
+  li.setAttribute('data-entry-id', entry.entryId);
```

Comment:
You shouldn't need to duplicate the attribute like this. I would just put the `data-entry-id` on the `li` and look up to the ancestor `li` for the data whenever you need to.